README.md

```diff
@@ -16,8 +16,10 @@ kalliope install --git-url https://github.com/kalliope-project/kalliope_neuron_o
 | api_key   | YES      | None    |                             | User API key of the OWM API                                                                       |
 | location  | YES      | None    |                             | The location                                                                                      |
 | lang      | No       | en      | multiple                    | First 2 letters cf : section Multilingual support in : [lang](https://openweathermap.org/current) |
-| temp_unit | No       | Kelvin  | Celsius, Kelvin, Fahrenheit |                                                                                                   |
-| country   | No       | US      | multiple                    |  Frist 2 letters of the country cf API doc                                                        |
+| temp_unit | No       | Kelvin  | Celsius, Kelvin, Fahrenheit |                                                            |                                                      
+| country   | No       | US      | multiple                    | Frist 2 letters of the country cf API doc                  |
```

Comment:
add a link the the API doc would be useful then.
Never seen that it was not present.

Reply:
Done